pages/get_involved.js

```diff
@@ -0,0 +1,139 @@
+  {
+    icon: <PlaceholderIcon2 />,
+    label: 'Fundraising',
+  },
+  {
+    icon: <PlaceholderIcon2 />,
+    label: 'Partnerships',
+  },
+];
+
+export default () => (
+  <div>
+    <Section theme="slate" title="You Can Make An Impact">
+      <div>
+        Whether you&apos;re interested in mentoring, supporting our operations, or simply donating
+        to help military veterans, service members, and spouses launch a career in software
+        development, Operation Code is always looking for supporters like you.
+      </div>
+    </Section>
+
+    <Section theme="mist" contentClassName={styles.alignCenter}>
+      <h4>The Power of Mentorship</h4>
+      <div className={styles.marginBottom}>
+        By mentoring one of our members, you will help them:
+      </div>
+      <div className={classNames(styles.flexDisplay, styles.marginBottom)}>
+        {mentorItems.map(item => (
+          <Badge key={item.label} icon={item.icon} label={item.label} />
+        ))}
+      </div>
+    </Section>
+
+    <Section theme="slate">
+      <div className={classNames(styles.cta, styles.ctaImageLeft)}>
+        <div className={styles.ctaImage}>
+          <img src={`${s3}stock_paired-programming.jpg`} alt="Empower Our Community" />
+        </div>
+        <div className={styles.ctaText}>
+          <h4>Empower Our Community</h4>
+          <p>
+            Make a difference in the lives of military veterans, service members, and spouses who
+            are eager to transition into a software development career.
+          </p>
+          <Button>Become a Mentor</Button>
+        </div>
+      </div>
+    </Section>
+
+    <Section theme="mist" contentClassName={styles.alignCenter}>
+      <h4>Ways You Can Support Operation Code</h4>
+      <div className={classNames(styles.marginBottom, styles.centerText)}>
+        We&apos;re always looking for volunteers who are dedicated to making an impact in the lives
+        of military veterans, service members, and spouses. You can help us with:
+      </div>
+      <div className={classNames(styles.flexDisplay, styles.marginBottom)}>
+        {supportItems.map(item => (
+          <Badge key={item.label} icon={item.icon} label={item.label} />
+        ))}
+      </div>
+    </Section>
+
+    <Section theme="slate">
+      <div className={classNames(styles.cta, styles.ctaImageRight)}>
+        <div className={styles.ctaImage}>
+          <img src={`${s3}stock_paired-programming.jpg`} alt="Support Our Mission" />
+        </div>
+        <div className={styles.ctaText}>
+          <h4>Support Our Mission</h4>
+          <p>
+            Make a difference in the lives of military veterans, service members, and spouses who
+            are eager to transition into a software development career.
+          </p>
+          <Button>Support OC</Button>
+        </div>
+      </div>
+    </Section>
+
+    <Section theme="mist" contentClassName={styles.alignCenter}>
+      <h4>Donate to Deploy the Future</h4>
```

Comment:
Instead of using the <h4> within the Section components, I'd prefer that you do:

```javascript
<Section {...thePropsYouAlreadyHave} title="Donate To Deploy The Future" hasHeadingLines={false}>
```

Reply:
Oh nice! I missed the fact that you could do that -- I'll change the rest of Sections that don't include a photo to use that. Although ideally for the two sections with a photo inserted... I'm guessing the Section component would need an optional prop to display a photo on the left or right?

Reply:
I think you can accomplish what you need with all of them!

```javascript
<Section
  {...thePropsYouAlreadyHave}
  className={styles.alignLeft}
  title="Donate To Deploy The Future"
  hasHeadingLines={false}
>
```

```css
.alignLeft {
  align-items: flex-start:
}
```

That may be it, but if it's not you can surely get the right CSS by passing className to the component.

Still not ideal, but let's push the abstraction of the Section component to a different PR

Reply:
Hmm I still think `Section` would need a headerClassName and/or have the HTML grouped differently for it to look right. I tried the above but but since `Header` in the `Section` spans across the whole div (whereas before the title was part of the content div and could be grouped into one column with flex), it still looks a bit off.

So I did change all the `Section` components to use `title` prop and just commented out the relative positioning of the photo for now. Otherwise it looked a bit strange with the offset... yeah, the editing of the Section component can be done in a different PR.

Reply:
If we really wanted to, we could beat the "specificity war" by applying a classname to the outer-most div of the page, but that's pretty silly.

I like your path better. Forget about raising the images for now.

I'm going to try to get to redoing the Section component tonight.

Assuming we merge this, and that by end of day, perhaps you can integrate the new components.

Reply:
Ohh I see what you mean. Okay! Yeah, I'm on the last thing which is figuring out the storybook file with knobs then I'll push the new changes.

Reply:
Highly recommend simply learning by example - check out other complete story files 👍 

If you'd like to completely dive into Storybook as a learning exercise (why it's useful, how to set it up, how to use it to upgrade your flow) check out learnstorybook.com